test/extended/deployments/deployments.go

```diff
@@ -385,6 +385,9 @@ var _ = g.Describe("[Feature:DeploymentConfig] deploymentconfigs", func() {
 				o.Expect(waitForLatestCondition(oc, "deployment-test", deploymentRunTimeout, deploymentReachedCompletion)).NotTo(o.HaveOccurred())
 
 				e2e.Logf("checking the logs for substrings\n%s", out)
+				if out == "<pod already deleted>" {
```

Comment:
deplyoer pod should never be deleted while issuing the logs call since we artificially prolong the deployer transition between Running and a terminal state by 15s - if that's not enough something got slower and we need to raise it or fix the performance degradation

Reply:
Where do we artificially prolong that?  Deployment isn't marked complete until hooks and deployment are complete, so by the time we get here we're racing.  The logs show that.